shell/platform/fuchsia/dart_runner/tests/fidl/flutter.example.echo/BUILD.gn

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 The Fuchsia Authors. All rights reserved.
```

Comment:
Replace with Flutter license, here and elsewhere in the PR.

For C++/Dart code:

```
// Copyright 2013 The Flutter Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
```

For other files:

```
# Copyright 2013 The Flutter Authors. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.
```

Not necessary for the `README.md`s.